tools/Dockerfile.CI

```diff
@@ -0,0 +1,26 @@
+FROM timbru31/java-node:11-alpine
+
+# Install Bazel
+#  Reference: https://github.com/davido/bazel-alpine-package)
+ADD https://raw.githubusercontent.com/davido/bazel-alpine-package/master/david@ostrovsky.org-5a0369d6.rsa.pub \
+    /etc/apk/keys/david@ostrovsky.org-5a0369d6.rsa.pub
+ADD https://github.com/davido/bazel-alpine-package/releases/download/0.26.1/bazel-0.26.1-r0.apk \
+    /tmp/bazel-0.26.1-r0.apk
+RUN apk add /tmp/bazel-0.26.1-r0.apk
```

Comment:
Does our build not work with bazel 1.0?

Reply:
I was acting fast to get bazel on an Alpine container and copied rules from elsewhere.  It will work with bazel 1; this is a good point. I'll work on an upgrade. 

Maybe in a later PR?

Reply:
1.1.0 would be even better :-)

Reply:
Broke out into issue #4025, will implement later.

Reply:
Actually, I think I got the latest bazel installed in this PR.